libs/cardano-ledger-test/src/Test/Cardano/Ledger/Alonzo/Tools.hs

```diff
@@ -1,9 +1,14 @@
 {-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DeriveAnyClass #-}
```

Comment:
Are all of these extensions needed? Shouldn't `StandaloneDeriving` be enough?

Reply:
@Jimbo4350 you are right. Thanks! :raised_hands: